docs/source/llms/tracing/index.rst

```diff
@@ -907,19 +904,50 @@ You can delete traces based on specific criteria using the :py:meth:`mlflow.clie
         experiment_id="1", max_timestamp_millis=current_time, max_traces=10
     )
 
-Setting and Deleting Trace Tags
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Trace Tags
+----------
+
+Tags can be added to traces to provide additional metadata at the trace level. For example, you can attach a session ID to a trace to group traces by a conversation session. MLflow provides APIs to set and delete tags on traces. Select the right API based on whether you want to set tags on an active trace or on an already finished trace.
+
+Setting Tags on an Active Trace.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
If we're marking the API as experimental, let's add a note here stating that the functionality may be expanded in the future (but not removed).

Reply:
Actually I don't think we need to explicitly mark this experimental. We may add some parameters but will not likely do backward incompatible change🙂